client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -484,6 +516,21 @@ export const SeoForm = React.createClass( {
 					</Notice>
 				}
 
+				{ conflictedSeoPlugin &&
+					<Notice
+						status="is-warning"
+						showDismiss={ false }
+						text={ translate(
+							'Your SEO settings are managed by the following plugin: %(pluginName)s',
+							{ args: { pluginName: conflictedSeoPlugin.name } }
+						) }
+					>
+						<NoticeAction href={ getPluginDetailUrl( conflictedSeoPlugin.slug, slug ) }>
+							{ translate( 'View Plugin' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'View Plugin Docs' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).